CONTRIBUTING.rst

```diff
@@ -16,6 +16,33 @@ useful:
 - give a minimal example demonstrating the bug,
 - copy and paste the error traceback.
 
+.. _making_test_files:
+
+Making test data files
+======================
+
+Test data fales are typically generated using third party software, for example using a proprietary
+software on a scientific instrument. These files are added to the `test suite <https://en.wikipedia.org/wiki/Test_suite>`_
+of RosettaSciIO to make sure that future code development will not introduce bug or feature 
+regression. It is important that the test data files area as small as possible to avoid to work
+with GBs large repository of test data. Indeed, the test suite is made of severals hundreds of
```

Comment:
```suggestion
regression. It is important that the test data files area as small as possible to avoid working
with a repository that contains GBs of test data. Indeed, the test suite is made of severals hundreds of
```